python/lsst/ap/verify/ap_verify.py

```diff
@@ -150,7 +151,15 @@ def runApVerify(cmdLine=None):
     log.debug('Command-line arguments: %s', args)
 
     workspace = WorkspaceGen3(args.output)
-    ingestDatasetGen3(args.dataset, workspace, args.namespace, args.restProxyUrl, processes=args.processes)
+    # Set the pipeline name as extra parameter for SasquatchDatastore to used
+    pipelineFile = _getPipelineFile(workspace, args)
+    extra = dict(vars(args)['extra'])
```

Comment:
can this fail with a KeyError if there is no `extra` in the `cmdLine`?

Reply:
I am verifying, but I think this is set to None if no values are provided. I will rectify if this is the case.